symplyphysics/definitions/thermodynamic_compressibility.py

```diff
@@ -16,49 +16,34 @@
 
 from sympy import Eq, Derivative
 from symplyphysics import (
-    units,
     Quantity,
-    Symbol,
-    Function,
     validate_input,
     validate_output,
+    symbols,
+    clone_as_function,
 )
 from symplyphysics.core.geometry.line import two_point_function, Point2D
 
-compressibility = Symbol("compressibility", 1 / units.pressure)
-r"""
-Compressibility of the gas.
-
-Symbol:
-    :code:`beta`
-
-Latex:
-    :math:`\beta`
+compressibility = symbols.thermodynamic_compressibility
 """
-
-volume = Function("volume", units.volume)
+:symbols:`thermodynamic_compressibility` of the gas.
 """
-Volume of the gas as a function of pressure and other parameters.
 
-Symbol:
-    :code:`V(p)`
+volume = clone_as_function(symbols.volume, display_symbol="V(p)")
 """
-
-pressure = Symbol("pressure", units.pressure)
+:symbols:`volume` of the gas as a function of pressure and other parameters.
 """
-Pressure in the gas.
 
-Symbol:
-    :code:`p`
+pressure = symbols.pressure
+"""
+:symbols:`pressure` in the gas.
 """
 
 definition = Eq(compressibility, -1 * Derivative(volume(pressure), pressure) / volume(pressure))
-r"""
-:code:`beta = -1 / V(p) * Derivative(V(p), p)`
+"""
+:laws:symbol::
 
-Latex:
-    .. math::
-        \beta = - \frac{1}{V(p)} \frac{\partial V}{\partial p}
+:laws:latex::
```

Comment:
Partials are not rendered properly. Some other parameters should exist in function to let latex generator know, that it is a partial derivative. Use manual formula description.